### DISCUSSION TITLE
I want student developer pack but i don't have id card can i upload my 12th admit card

### DISCUSSION THREAD

**Discussion by abhishek-nexgen-dev** [community/community#138949] [GitHub Education] [answered]
### Hi everyone,

I want student developer pack but i don't have id card can i upload my 12th admit card

**Comment by samueltexa** [comment]
Hi @Code-With-Abhishek-Kumar,
Does your  12th-grade admit card include your current enrollment status?????.

**Reply by abhishek-nexgen-dev** [reply, marked as answer]
> current enrollment status?????.

 Yes, the JAC (Jharkhand Academic Council) 12th-grade admit card typically includes important details about the student, such as:

    Name
    Roll number
    School name
    Subjects
    Enrollment status
